### ISSUE TITLE
Readme info is sparse

### ISSUE THREAD

**Issue by JayHoltslander** [JeffBrockStudio/_jbs#1]
Readme says this is based on [_tk](https://github.com/Themekraft/_tk) which appears to be the same as this but more recently updated. Can you clarify how _jbs is different than _tk ?

**Comment by JeffBrockStudio** [comment]
Hi Jay! Thanks for the heads up -- the primary difference is that _jbs uses the official SASS port of Bootstrap, rather than the LESS version. I'll update the Readme to reflect this.

**Comment by JayHoltslander** [comment]
I thought _tk was also Sass. Hmm. Will look again.

**Comment by JeffBrockStudio** [comment]
If I'm mistaken, please let me know.

**Comment by JeffBrockStudio** [comment]
Yikes, I think you're right! https://github.com/Themekraft/_tk/tree/master/includes/resources/bootstrap/css I may take my repo private, as this would mean there's not enough of a difference to make it valuable to the public.

**Comment by JayHoltslander** [comment]
Yeah and it looks like they're pretty actively maintaining the Sass port too. 
Ok. Was just wondering if maybe I'd missed seeing some other differentiator.
